library/task-library/K8 Module Escripts/List_namespaces.py

```diff
@@ -0,0 +1,24 @@
+#script
+
+#Variables used in this script 
+KUBERNETES_CLUSTER_IP=""
+API_KEY=""
+
+import kubernetes.client
+from kubernetes.client.rest import ApiException
```

Comment:
same as above, these modules can't be imported in escript

Reply:
Same as above